helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java

```diff
@@ -67,6 +67,7 @@
  *   },
  *   "id" : "myservice:0.5:3539b7bc2235d53f79e6e8511942bbeaa8816265",
  *   "image" : "myregistry:80/janedoe/myservice:0.5-98c6ff4",
+ *   "hostname": "myhost",
```

Comment:
Can you also update this part of the docs? https://github.com/spotify/helios/blob/master/docs/user_manual.md#using-a-helios-job-config-file